stream-chat-android/src/main/java/com/getstream/sdk/chat/utils/Linkify.kt

```diff
@@ -0,0 +1,212 @@
+package com.getstream.sdk.chat.utils
+
+import android.annotation.SuppressLint
+import android.text.Spannable
+import android.text.SpannableString
+import android.text.Spanned
+import android.text.method.LinkMovementMethod
+import android.text.style.URLSpan
+import android.text.util.Linkify
+import android.widget.TextView
+import androidx.core.util.PatternsCompat
+import java.util.ArrayList
+import java.util.Collections
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
+/**
+ * Linkify links the part of the text based on matching pattern.
+ *
+ * This class is a simplified version of [Linkify] and differs only in one following way
+ * It doesn't remove any existing URLSpan from the Spannable.
+ */
+internal object Linkify {
```

Comment:
What do you think about moving it to `ui-common` module? We could mark it with @InternalStreamApi annotation and reuse it in both `ui` modules

Reply:
You mentioned that "we are going to remove ui-common sometime". Is it still a good idea to do that?

Reply:
We don't know when we are going to do that and it should be fine because this object shouldn't be used outside our SDK